akka-docs/rst/scala/persistence-query.rst

```diff
@@ -0,0 +1,212 @@
+.. _persistence-query-scala:
+
+#################
+Persistence Query
+#################
+
+Akka persistence query complements :ref:`persistence-scala` by providing an universal asynchronous stream based
+query interface that various journal plugins can implement in order to expose their query capabilities.
+
+The most typical use case of persistence query is implementing the so-called query side (also known as "read side"),
+in the popular CQRS architecture pattern - in which the writing side of the application (e.g. implemented using akka
+persistence) is completely separated from the query side which can be implemented using akka persistence query.
+
+While queries may be performed directly on the same datastore, it is also a very common pattern to use the queries
+to create projections of the write-side's events and store them into a separate journal which is optimised for more
+complex queries - this migration of data, possibly including some transformation along the way, can be easily implemented
+using akka persistence query as well - by pulling out of one Journal and storing into another one.
+
+.. warning::
+
+  This module is marked as **“experimental”** as of its introduction in Akka 2.4.0. We will continue to
+  improve this API based on our users’ feedback, which implies that while we try to keep incompatible
+  changes to a minimum the binary compatibility guarantee for maintenance releases does not apply to the
+  contents of the ``akka.persistence.query`` package.
+
+Design overview
+===============
+
+Akka persistence query is purposely designed to be a very loose API.
```

Comment:
True (must have been thinking of windsurfing https://en.wikipedia.org/wiki/Shaka_sign, hang loose) ;-)